hugo/layouts/partials/social/author_social.html

```diff
@@ -7,6 +7,3 @@
 {{ if .Params.instagram }}
 <a class="-social-link" href="https://www.instagram.com/{{ .Params.instagram }}" target="_blank"><i class="fa fa-instagram"></i></a>
 {{ end }}
-{{ if .Params.instagram }}
```

Comment:
I think we'll probably want to keep this block for anyone who may have set their `linkedin` account. We'll just instead want to change the conditional to `{{ if .Params.linkedin  }}`